setup.py

```diff
@@ -13,6 +13,7 @@
                 'partd >= 0.3.7', 'cloudpickle >= 0.2.1'],
   'distributed': ['distributed >= 1.16', 's3fs >= 0.0.8'],
   'delayed': ['toolz >= 0.7.3'],
+  'stats': ['scipy >= 0.19.0']
```

Comment:
Hrm, I'm tempted to leave this out, just so that `pip install dask[complete]` doesn't rely on `scipy`.  Is this still a pain to pip install or have those days passed?